### PR TITLE
BaSP-TG-80: Form tasks

### PR DESCRIPTION
**Pull request by mara666**
- Replace relative paths with absolute paths
- Implement Input component
- Implement Joi validations
- Use joiResolver() from React hook forms
- Create a file named validations.jsx inside Form folder and create the schema using Joi inside
- Implement React hook forms

### CODE REVIEW

File: src/Components/Tasks/Form/index.jsx [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 const Form = () => {
-  const [inputValue, setInputValue] = useState({
-    description: ''
-  });
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+    reset
+  } = useForm({ mode: 'onBlur', resolver: joiResolver(taskSchema) });
```
[the remark this answers is not in the document]
Comment (mara666) [reply src/Components/Tasks/Form/index.jsx:17]: Okey, thank you for the suggest!

File: src/Components/Tasks/Form/validations.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export const taskSchema = Joi.object({
+  description: Joi.string()
+    .required()
+    .trim()
+    .min(3)
+    .max(100)
+    .pattern(/^(?=.[a-zA-Z].)([\w\s\W]+)$/)
+    .messages({
+      'string.empty': `"Description" can't be empty`,
```
Comment (Maxi-SR) [inline src/Components/Tasks/Form/validations.jsx:11]: You can use single quotes instead of backticks

Comment (mara666) [reply src/Components/Tasks/Form/validations.jsx:11]: okey! I fix it

**Review by PilarFernandezC — APPROVED** [review]
Looks good!

**Review by Juanoauil — APPROVED** [review]

**Pull request merged**